LDAR_Sim/model_code/ldar_sim_main.py

```diff
@@ -25,13 +25,34 @@
 import datetime
 import warnings
 import multiprocessing as mp
+import boto3 # for downloading data from AWS
 
 if __name__ == '__main__':
     # ------------------------------------------------------------------------------
     # -----------------------------Global parameters--------------------------------
     wd = "../inputs_template/"
     program_list = ['P_ref','P_alt', 'P_alt2', 'P_cont']  # Programs to compare; Position one should be the reference program (P_ref)
 
+    #-------------------------------------------------------------------------------
```

Comment:
@AkaMrmuffin This looks great - just a few notes:

1. Would it be possible to move this function to "general functions" and then just call it here with the ~2 lines? I think it's important to keep the ldar_sim_main file as "clean" as possible.
2. Can we read the era file name (era_file) from the input parameters (text files for each program that should contain weather data name) so that it doesn't have to be input here (and so that we include it as an input for each program instead)?

Reply:
I've moved this function to the general functions file. The inputs now change to the path of the working directory and a string that stands for location. For example "AB" stands for Alberta. I also added some logic into the function to check multiple conditions.